readthedocs/doc_builder/constants.py

```diff
@@ -26,51 +26,6 @@
     )
     DOCKER_IMAGE_SETTINGS.update(old_config)
 
-DOCKER_LIMITS = {
-    'memory': '200m',
-    'time': 600,
-}
-
-# Set docker limits dynamically based on system memory
-# This assumes 1-builder per server
-try:
-    total_memory = int(subprocess.check_output("free -m | awk '/^Mem:/{print $2}'", shell=True))
-except ValueError:
-    # On systems without a `free` command it will return a string to int and raise a ValueError
-    log.exception('Failed to get memory size. Using defaults docker limits')
-    total_memory = 0
-
-if total_memory > 14000:
-    DOCKER_LIMITS.update({
-        'memory': '13g',
-        'time': 2400,
-    })
-elif total_memory > 8000:
-    DOCKER_LIMITS.update({
-        'memory': '7g',
-        'time': 1800,
-    })
-elif total_memory > 7000:
-    # This is to catch AWS instances that actually only have 7.5G memory
-    DOCKER_LIMITS.update({
-        'memory': '6g',
-        'time': 1800,
-    })
-elif total_memory > 4000:
-    DOCKER_LIMITS.update({
-        'memory': '3g',
-        'time': 900,
-    })
-elif total_memory > 2000:
-    DOCKER_LIMITS.update({
-        'memory': '1g',
-        'time': 600,
-    })
-
-if hasattr(settings, 'DOCKER_LIMITS'):
-    DOCKER_LIMITS.update(settings.DOCKER_LIMITS)
```

Comment:
We should keep this logic, the current PR doesn't let users override the DOCKER_LIMITS.